inert.js

```diff
@@ -110,6 +110,24 @@ class InertRoot {
    */
   _makeSubtreeUnfocusable(startNode) {
     composedTreeWalk(startNode, (node) => { this._visitNode(node); });
+
+    let activeElement = document.activeElement;
+    if (!document.contains(startNode)) {
```

Comment:
I can't imagine it being significantly more or less expensive than a plain treewalk.